tests/integration/targets/content_library/tasks/main.yaml

```diff
@@ -0,0 +1,102 @@
+---
+- set_fact:
+    datastore_name: "rw_datastore"
+    local_library_name: "local_library_001"
+
+- name: Get Datastore id required for local library creation
+  vmware.vmware_rest.vcenter_datastore_info:
+    filter_names:
+      - "{{ datastore_name }}"
+  register: r
```

Comment:
I use `result` in all the other tests. Do you mind if we use the same variable name in this playbook too?

Reply:
OK.